openmetadata-ui/src/main/resources/ui/src/components/DataInsightDetail/DataInsightSummary.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Card, Col, Row, Typography } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { OVERVIEW } from '../../pages/DataInsightPage/DataInsight.mock';
+
+const DataInsightSummary = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Card data-testid="summary-card">
+      <div data-testid="summary-card-heder">
```

Comment:
Why do we need this extra div?
Can we use title prop of card instead?

Reply:
Sure.